src/legacy/audio_ail.go

```diff
@@ -271,7 +268,7 @@ func Sub_43D2D0() {
 }
 
 func Sub_486620(v unsafe.Pointer) {
```

Comment:
```suggestion
func Sub_486620(v *TimerGroup) {
```
Since type layout is known, it's better to start passing it around to unsure type safety on Go side at least. Or is it too invasive to do at once?